Makefile

```diff
@@ -131,8 +131,13 @@ cleanup-control-plane:
 .PHONY: cleanup-partition
 cleanup-partition:
 	mkdir -p clab-mini-lab
+
 	sudo $(CONTAINERLAB) destroy --topo mini-lab.cumulus.yaml
+
+# destroying the sonic lab requires the image to exist, otherwise it fails with bind path verification
+	touch sonic-vs.img
 	sudo $(CONTAINERLAB) destroy --topo mini-lab.sonic.yaml
+	rm -f sonic-vs.img
```

Comment:
Using an conditional?
```
ifneq ($(wildcard sonic-vs.img),)
	sudo $(CONTAINERLAB) destroy --topo mini-lab.sonic.yaml
endif
```

Reply:
Maybe its easier to create a container image that contains the VM image. Let me try something next week.

Reply:
Have you come up with something better? Release pipeline is currently broken due to cleanup problems so we should find something soon.

Reply:
I will look into it during my journey.

Reply:
We prioritized fixing the pipeline for now. If you find some time to improve the cleanup please feel free to raise a new pull request.